src/include/homestore/btree/detail/btree_mutate_impl.ipp

```diff
@@ -292,7 +266,7 @@ btree_status_t Btree< K, V >::split_node(const BtreeNodePtr& parent_node, const
     child_node2->set_next_bnode(child_node1->next_bnode());
     child_node1->set_next_bnode(child_node2->node_id());
     child_node2->set_level(child_node1->level());
-    uint32_t child1_filled_size = m_bt_cfg.node_data_size() - child_node1->available_size();
+    uint32_t child1_filled_size = child_node1->node_data_size() - child_node1->available_size();
```

Comment:
Given that node now stores the data_size in its header, it doesn't need every call needs to pass and get this information from m_bt_cfg. Saves a lot of ugly passing of config references to node level code.